{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2024-31879",
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "state": "PUBLISHED",
        "assignerShortName": "ibm",
        "dateReserved": "2024-04-07T12:44:32.087Z",
        "datePublished": "2024-05-18T15:40:03.867Z",
        "dateUpdated": "2024-07-05T15:34:55.857Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "cpes": [
                        "cpe:2.3:a:ibm:i:7.2:*:*:*:*:*:*:*",
                        "cpe:2.3:a:ibm:i:7.3:*:*:*:*:*:*:*",
                        "cpe:2.3:a:ibm:i:7.4:*:*:*:*:*:*:*"
                    ],
                    "defaultStatus": "unaffected",
                    "product": "i",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "7.2, 7.3, 7.4"
                        }
                    ]
                }
            ],
            "descriptions": [
                {
                    "lang": "en",
                    "supportingMedia": [
                        {
                            "base64": false,
                            "type": "text/html",
                            "value": "IBM i 7.2, 7.3, and 7.4 could allow a remote attacker to execute arbitrary code leading to a denial of service of network ports on the system, caused by the deserialization of untrusted data.  IBM X-Force ID:  287539."
                        }
                    ],
                    "value": "IBM i 7.2, 7.3, and 7.4 could allow a remote attacker to execute arbitrary code leading to a denial of service of network ports on the system, caused by the deserialization of untrusted data.  IBM X-Force ID:  287539."
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "NONE",
                        "baseScore": 7.5,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N",
                        "version": "3.1"
                    },
                    "format": "CVSS",
                    "scenarios": [
                        {
                            "lang": "en",
                            "value": "GENERAL"
                        }
                    ]
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-502",
                            "description": "CWE-502 Deserialization of Untrusted Data",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm",
                "dateUpdated": "2024-05-22T02:31:05.087Z"
            },
            "references": [
                {
                    "tags": [
                        "vendor-advisory"
                    ],
                    "url": "https://www.ibm.com/support/pages/node/7154380"
                },
                {
                    "tags": [
                        "vdb-entry"
                    ],
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/287539"
                }
            ],
            "source": {
                "discovery": "UNKNOWN"
            },
            "title": "IBM i denial of service",
            "x_generator": {
                "engine": "Vulnogram 0.2.0"
            }
        },
        "adp": [
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "id": "CVE-2024-31879",
                                "role": "CISA Coordinator",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "yes"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "version": "2.0.3",
                                "timestamp": "2024-05-20T17:48:44.436939Z"
                            }
                        }
                    }
                ],
                "providerMetadata": {
                    "orgId": "8c464350-323a-4346-a867-fc54517fa145",
                    "shortName": "CISAADP",
                    "dateUpdated": "2024-05-23T01:31:50.512Z"
                }
            },
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-07-05T15:34:55.857Z"
                },
                "references": [
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/7154380/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/287539/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "vendor-advisory",
                            "x_snapshot"
                        ],
                        "url": "https://www.ibm.com/support/pages/node/7154380/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "vdb-entry",
                            "x_snapshot"
                        ],
                        "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/287539/adp-test-6-28"
                    }
                ]
            }
        ]
    }
}